avocado/core/suite.py

```diff
@@ -24,13 +24,15 @@ class TestSuiteStatus(Enum):
 
 
 class TestSuite:
-    def __init__(self, name, config, tests=None):
+    def __init__(self, name, config, tests=None, job_config=None):
```

Comment:
What if the `TestSuite` takes any number of configurations, and doesn't treat the first as its own and the second as a job config?  I mean, the resulting configuration would always follow the pattern of last updates the second last, etc.

I can say that I'd like the method signature less... but still would like to let you know about the idea.

Reply:
Yes, it is a valid alternative.

But I confess that I prefer how it is today because of the explicitly. We are explicitly making reference to the job_config. I guess that from the user perspective will be easier to understand.

What do you think? Can we keep like that?